service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/transport/amqps/AmqpFileUploadNotificationReceive.java

```diff
@@ -76,16 +95,73 @@ public AmqpFileUploadNotificationReceive(String hostName, String userName, Strin
         this.sslContext = sslContext;
     }
 
+    public AmqpFileUploadNotificationReceive(
+            String hostName,
+            TokenCredential authenticationTokenProvider,
```

Comment:
I believe we will rename this to credential everywhere as per discussion on my PR?

Reply:
Yep, I will rename this to "TokenCredential credential"